### ISSUE TITLE
TypeError: unsupported operand type(s) for +: 'PosixPath' and 'str'

### ISSUE THREAD

**Issue by isidentical** [edgedb/edgedb-python#120]
```
(.venv) isidentical@search-tree-science-1:~/reiz.io$ ./scripts/regen_db.sh 
Stopping the server...
Re-starting the server...
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/isidentical/reiz.io/reiz/db/reset.py", line 49, in <module>
    main()
  File "/home/isidentical/reiz.io/reiz/db/reset.py", line 45, in main
    drop_and_load_db(**vars(options))
  File "/home/isidentical/reiz.io/reiz/db/reset.py", line 25, in drop_and_load_db
    with closing(edgedb.connect(dsn, database="edgedb")) as connection:
  File "/home/isidentical/reiz.io/.venv/lib/python3.8/site-packages/edgedb/blocking_con.py", line 244, in connect
    addrs, params, config = con_utils.parse_connect_arguments(
  File "/home/isidentical/reiz.io/.venv/lib/python3.8/site-packages/edgedb/con_utils.py", line 339, in parse_connect_arguments
    addrs, params = _parse_connect_dsn_and_args(
  File "/home/isidentical/reiz.io/.venv/lib/python3.8/site-packages/edgedb/con_utils.py", line 202, in _parse_connect_dsn_and_args
    path = (pathlib.Path.home() /
TypeError: unsupported operand type(s) for +: 'PosixPath' and 'str'

```

Possible fix:
```diff
+++ b/edgedb/con_utils.py
@@ -200,7 +200,7 @@ def _parse_connect_dsn_and_args(*, dsn, host, port, user,
                 f"nor valid instance name"
             )
         path = (pathlib.Path.home() /
-                '.edgedb' / 'credentials' / dsn + '.json')
+                '.edgedb' / 'credentials' / dsn).with_suffix('.json')
         try:
             creds = credentials.read_credentials(path)
         except Exception as e:
```

**Comment by MNouman313** [comment]
![error](https://user-images.githubusercontent.com/76149323/105162278-35902c00-5b34-11eb-91e8-7cbfd628c02e.png)
It gives me an error like:
TypeError: unsupported operand type(s) for +: 'WindowsPath' and 'str'

**Comment by Fogapod** [comment]
Looks like this has been fixed by #121: https://github.com/edgedb/edgedb-python/commit/dc532eeb308b8c41c41c2913aeb5fb84ab5ad045#diff-74188b7ca9fbcd019d3646664133b00e705e3453eb08dfc1726dde21a1e29f3bR207

**Comment by elprans** [comment]
Yep, this is fixed.
